.stylelintrc

```diff
@@ -1,51 +1,80 @@
 {
   "extends": [
-    "stylelint-config-standard",
-    "stylelint-config-suitcss"
+    "stylelint-config-suitcss",
+    "stylelint-config-standard-scss"
```

Comment:
Do we need both? In addons-blog, I only use `styleint-config-standard-scss` FWIW: https://github.com/mozilla/addons-blog/pull/338/files

Reply:
I'm not sure why we need this, but I didn't want to remove something that was there when I arrived. Now that you've asked the question, I took a look and it seems the only thing we use `suitCSS` for is our selector naming conventions, and I ended up overriding `stylelint-config-standard-scss`'s rule with a regex that supports our convention. So I think we can remove this.